projects/igniteui-angular/src/lib/grids/filtering/grid-filtering-row.component.ts

```diff
@@ -300,6 +299,8 @@ export class IgxGridFilteringRowComponent implements AfterViewInit {
      */
     public onInputClick() {
         if (this.column.dataType === DataType.Boolean) {
+            this.input.nativeElement.blur();
```

Comment:
Same as above - calling the blur() method is not necessary, focusing the prefix will trigger it anyway.

Reply:
@SAndreeva the blur() is removed